docs/reference-guides/filters/global-styles-filters.md

```diff
@@ -1,11 +1,11 @@
 # Global Styles Filters
 
-WordPress 6.1 has introduced some server-side filters to hook into the data provided to Global Styles & Settings:
```

Comment:
This only updates the names of the filters. I'm undecided as to whether we should rename the "Global Styles Filters" page to "Theme.json Filters". Because it's [published](https://developer.wordpress.org/block-editor/reference-guides/filters/global-styles-filters/) to the block editor handbook already, renaming it requires asking someone from the meta team to remove the old one. As it is also works fine if we want to add new filters later, so I lean towards maintaining what we have.